test/eredis_tests.erl

```diff
@@ -3,7 +3,7 @@
 -include_lib("eunit/include/eunit.hrl").
 -include("eredis.hrl").
 
--define(?ASSERT_RECEIVE(Pattern),
```

Comment:
Assuming that you didn't, on a later commit, change whatever uses there are of this renamed macro, does it mean that it's not actually being used?

If that's the case, then we should remove its declaration (except for some extraordinary circumstance, yet unknown by me, that calls for its existence.)

Reply:
Perhaps you fixed it later? Please resolve if that's the case. I'm sorry, I don't think this commit-by-commit approach to reviews works very well for me...

Reply:
Nothing to see here.